db/seeds.d/02-seeds.rb

```diff
@@ -0,0 +1,161 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```

Comment:
If we are breaking seeds up, can we break up the rest of this file into logical seed files?